draft-ietf-mmusic-dtls-sdp.xml

```diff
@@ -368,10 +386,12 @@
             <t>
                 If the offerer inserts the SDP 'setup' attribute with an 'actpass' or 'passive' attribute value, the
                 offerer MUST be prepared to receive a DTLS ClientHello message (if a new DTLS association
-                is established by the answerer) from the answerer before the offerer receives the SDP answer.
+                is established by the answerer) from the answerer before the offerer receives the SDP answer. In such
+                case the offerer MAY cache the ClientHello message, but the offerer MUST NOT initiate a DTLS

```

Comment:
> If solution is designed to exchange transport parameters, why fingerprints and tls-id cannot be exchanged at the same time? 

Because a protocol could exist where the offerer doesn't need any transport parameters from the answerer, and thus there wouldn't be any convenient place to stick the fingerprint/tls-id.

> The reason I am saying this is the more end point does with unverified party (proceed with DTLS handshake, receive media, decode media, cache received media), the more it allows for security issues or more resources it uses which provides opportunity for DOS attack.

This is a good argument. But couldn't the choice to take this risk or not be left to the implementer?